tests/annotations/tests.py

```diff
@@ -636,7 +636,7 @@ def test_annotation_exists_aggregate_values_chaining(self):
         ])
 
     @skipIf(
-        connection.vendor == 'mysql' and 'ONLY_FULL_GROUP_BY' in connection.sql_mode,
+        connection.vendor == 'mysql',
```

Comment:
Doesn't this mean, the tests are skipped all the time on mysql, not only when the `ONLY_FULL_GROUP_BY` is used? How about something like this:
```suggestion
        connection.vendor == 'mysql' and 'ONLY_FULL_GROUP_BY' in getattr(connection, "sql_mode", set()),
```
or whatever type `sql_mode` usually has.

Reply:
We added `sql_mode` in a19505eb2ecd4a07ff39f3f66898e68b69d0e680, it doesn't exists in Django 3.0. I think it's fine to always skip this test on MySQL because we use `ONLY_FULL_GROUP_BY` in CI.

Reply:
Your suggestion will cause a failure in CI (see ticket-31331), because `'ONLY_FULL_GROUP_BY' in getattr(connection, "sql_mode", set())` will always be false in Django 3.0.

Reply:
> We added `sql_mode` in [a19505e](https://github.com/django/django/commit/a19505eb2ecd4a07ff39f3f66898e68b69d0e680), it doesn't exists in Django 3.0. I think it's fine to always skip this test on MySQL because we use `ONLY_FULL_GROUP_BY` (default in MySQL 5.7+) in CI.

Ah, okay. I wasn't aware of that.